db/seeds.rb

```diff
@@ -1,7 +1,40 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+
+user1 = User.find_by(email: "booklover@example.com")
+unless user1.present?
```

Comment:
You could make the `seeds` script idempotent so you don't have to worry about these `unless` conditions. i.e. delete all the records all the beginning and then re-create what you want. That way you can run the seeds again and again. (This isn't always the right approach, depending on what you're doing, but since I saw the `unless` pattern, I thought I should mention it).